packages/tools/src/store/ToolGroupManager/ToolGroup.ts

```diff
@@ -669,6 +682,38 @@ export default class ToolGroup implements IToolGroup {
     return cloneDeep(_configuration);
   }
 
+  public clone({ newToolGroupId, fnToolFilter = null }) {
```

Comment:
require doc

Reply:
I prefer toolGroupId and you rename it yourself in the props to newToolGroupId.
fnToolFilter not clear as there is no doc